package.json

```diff
@@ -12,6 +12,9 @@
   "dependencies": {
     "eth-provider": "^0.2.2",
     "ethers": "^4.0.27",
+    "express": "^4.17.1",
```

Comment:
Why do we have to import express, when hubot comes with express by default?

Reply:
Whoops, that's a leftover of something I tried to fix a problem.

Reply:
Correction: when removing that line, I actually get an error when starting HAL8000

> Error: Cannot find module 'express'